data-access/src/functions/timer-gl-daily-summary/gl-transaction.ts

```diff
@@ -1,13 +1,36 @@
+import { GlDailySummaryProcessorImpl } from "../../../seedwork/services-seedwork-finance/gl-daily-summary-processor";
+import { GlDailySummaryStorageProviderBlobImpl } from "../../../seedwork/services-seedwork-finance/gl-daily-summary-storage-provider-blob";
+import { DateUtils } from "../../../seedwork/utils/date-utils";
 import { TimerContext } from "../core/timer.context-builder";
 
 export const ProcessGLTransactions = async (context: TimerContext): Promise<void> => {
   try {
-    console.log('[TestTimerHandler] | Invocation ID ', context);
-    console.log('[TestTimerHandler] | Timer ', context);
+    console.log(`timer-gl-daily-summary | start | ${context}`);
     // Timer function logic here
+
+    const blobStorageProvider = GlDailySummaryStorageProviderBlobImpl.getInstance();
+    const lastProcessedGlTransactionDate = await blobStorageProvider.getLastProcessedGlTransactionDate();
+    const processingStartTimestamp: Date = lastProcessedGlTransactionDate ? 
+        DateUtils.convertToStartOfDayEST(DateUtils.addDaysToDate(lastProcessedGlTransactionDate.date, 1)) 
+        : new Date(Date.now() - 86400000);
```

Comment:
The first time the job runs will be the day the system goes live, there are no existing transaction summaries to refer to so we start looking at transactions the day before. This will ensure that all transactions will be recorded by subsequent runs of the timer job